data/json/vehicleparts/vehicle_parts.json

```diff
@@ -602,8 +602,8 @@
     "damage_modifier": 80,
     "durability": 100,
     "description": "A small fridge.  When turned on, it will cool the food inside, extending the time until the food spoils.",
-    "//": "Use average consumption, not the max on the appliance rating plate. 30W ~ 260kWh per annum",
-    "epower": "-30 W",
+    "//": "Use average consumption, not the max on the appliance rating plate. For big fridge 30W ~ 260kWh per annum is good, but for mini variant with decrased volume by ~ 6 time, surface area ~ 3-4 times and insulation thickness only by 2 time 10W is good too and very close to real world variant ",
+    "epower": "-10 W",
```

Comment:
24W comnsumption is good for very cheap fridge
https://subcold.com/products/75-litre-mini-fridge-eco75 - here is 11.8W variant
https://www.amazon.com/DOMETIC-75-Liter-Portable-Refrigerator-Freezer/dp/B09FS4YJ8F here is 10W
And there was two very cheap 24W variants
Make at leat 15W for minifridges, because it will be ~ middle value of bad and good mini fridges